extensions/extensions-helm/aissemble-configuration-store-chart/values.yaml

```diff
@@ -0,0 +1,35 @@
+# Default values for config-store-service
+app:
+  # Sets labels for app.kubernetes.io/name Default is Chart.name (aissemble-store-chart)
+  name: ''
+  # Sets the labels for app.kubernetes.io/version. Default is Chart.appVersion
+  version: ''
+
+hostname: configuration-store
+image:
+  name: boozallen/aissemble-configuration-store
+  imagePullPolicy: Always
+  dockerRepo: ghcr.io/
+  tag: "1.8.0-SNAPSHOT"
```

Comment:
C: We should avoid hard-coding this value.  We should be able to just leave it blank and default it to `Chart.AppVersion` in the templates. E.g. `{ Values.image.tag | default Chart.AppVersion }`. Otherwise this will have to be updated during each release and could be missed causing the version to be out of sync.

Reply:
Yeah thats a great point!